src/Context/ApiContext.php

```diff
@@ -194,23 +194,22 @@ public function determineBaseUri()
     }
 
     /**
-     * @return string
      * @throws BunqException
+     * @return string
      */
     private function determineBaseUriString()
     {
         if ($this->environmentType->getChoiceString() === BunqEnumApiEnvironmentType::CHOICE_PRODUCTION) {
             return self::BASE_URL_PRODUCTION;
         } elseif ($this->environmentType->getChoiceString() === BunqEnumApiEnvironmentType::CHOICE_SANDBOX) {
             return self::BASE_URL_SANDBOX;
-        } else {
-            throw new BunqException(
+        }
+        throw new BunqException(
```

Comment:
We always separate throws and returns (and also all the control structures) by one newline from the the rest of the code on the same indentation level. Let's please keep to it.